openapi/openapi.go

```diff
@@ -54,21 +62,20 @@ func SetRenderer(api *OpenAPI, renderer Renderer) {
 	})
 }
 
-// resolveRefs walks the tree and
-func resolveRefs(key string, parent, child Traversable) (Traversable, error) {
-	child.setParent(parent)
-	child.setName(key)
-	ref := child.getRef()
-	if ref != "" {
-		basePath := child.getBasePath()
-		ref := filepath.Base(child.getRef())
-		err := readRef(filepath.Join(basePath, ref), child)
+// resolveRefs calls readRef on references with the ref path modified appropriately for it's use
+func resolveRefs(key string, parent, node Traversable) (Traversable, error) {
+	node.setParent(parent)
```

Comment:
child might not be the best name, but `node` shadows a pretty crucial type in our codebase. I'd recommend calling it something more specific so that we can still refer to the node type it we need to and the reader clearly understands which type is being used.